sbt-bridge/src/xsbt/AbstractZincFile.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Zinc - The incremental compiler for Scala.
+ * Copyright Lightbend, Inc. and Mark Harrah
+ */
+
+package xsbt;
+
+import dotty.tools.io.AbstractFile;
+import xsbti.PathBasedFile;
+import xsbti.VirtualFile;
+
+import java.io.IOException;
+
+interface AbstractZincFile {
```

Comment:
Is this extra abstraction needed? `of` is a static method and could be anywhere else. `String id()` is already defined in `VirtualFileRef`, so we could use that instead of `AbstractZincFile`, or are the other methods in `VirtualFileRef` problematic? If so, it seems that `AbstractZincFile` should be defined in zinc itself.